forge-game/src/main/java/forge/game/ability/effects/RegenerationEffect.java

```diff
@@ -25,6 +30,7 @@ public void resolve(SpellAbility sa) {
             c.setDamage(0);
             c.setHasBeenDealtDeathtouchDamage(false);
             c.tap(true, cause, c.getController());
```

Comment:
Shouldn't tap return bool to check if it has changed?

Reply:
Currently Card.tap is void – want me to refactor it to boolean?

Reply:
Yeah boolean would be better to return if has really changed

Reply:
Oops thought I pushed this update